pkg/kubelet/dockershim/docker_container.go

```diff
@@ -144,8 +144,13 @@ func (ds *dockerService) CreateContainer(podSandboxID string, config *runtimeapi
 	}
 
 	// Apply Linux-specific options if applicable.
+	apparmorProfile := ""
 	if lc := config.GetLinux(); lc != nil {
 		// Apply resource options.
+		if lc.SecurityContext != nil && lc.SecurityContext.ApparmorProfile != "" {
```

Comment:
I think this should be in `applyContainerSecurityContext`, since it's part of the `SecurityContext`